contents/tutorials/css-selectors-for-actions.md

```diff
@@ -127,15 +127,15 @@ In general, CSS selectors are well-suited for situations such as...
 There are two ways to do it - through the Toolbar and Data Management section.
 
 ### The toolbar
-Go to your PostHog instance and select `Toolbar` in the bottom of the sidebar.
+Go to your PostHog instance and select `Launch toolbar` in the bottom of the sidebar.
 
-![Toolbar Location](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/toolbar_location.png)
+![Toolbar Location](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/toolbar_location.mp4)
 
 Then, press the `Open with Toolbar` button, which will take you to your website. You can read more about working with the toolbar [here](/manual/toolbar).
 
-After your website is open, you should see a PostHog logo 🦔 floating around. Press it. You will see a toolbar like the picture below.
+After your website is open, you should see a PostHog toolbar:
 
-![Toolbar In Action](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/toolbar_in_action.png).
+![Toolbar In Action](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/toolbar_in_action.png)
 
```

Comment:
This entire tutorial's screenshots are out of date + the tutorial itself could use some updating, but I don't think its a good use of time at the moment. 

Perhaps a good point of discussion at our next meeting on what to do with outdated content

Reply:
Agree